### PR TITLE
Clarify versioning policy

### PR DESCRIPTION
**Pull request by hadley**
i.e. that it applies to r-lib too.

For https://github.com/r-lib/evaluate/issues/173

### CODE REVIEW

File: content/blog/2019/r-version-support/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ rmd_hash: efe6168c59ef5121
 
 Though package-specific minimum versions of R are given in the `Depends` fields of individual `DESCRIPTION` files, our general policy is to support the **current version**, the **devel version**, and **four previous versions** of R.
 
-**Edit 2022-11-15[^1]**
+This policy applies to all packages that we support, i.e. as well as all packages in the tidyverse, it also applies the infrastructure packages that we maintain in [r-lib](https://github.com/r-lib).
```
Comment (jennybc) [inline content/blog/2019/r-version-support/index.md:22]:
```suggestion
This policy applies to all packages that we support, i.e. as well as all packages in the tidyverse, it also applies to the infrastructure packages that we maintain in [r-lib](https://github.com/r-lib).
```

File: content/blog/2019/r-version-support/index.md [hunk 2]
```diff
@@ -44,7 +44,9 @@ Tidyverse minimum R version support
 
 </div>
 
-[^1]: Add table with annual historic and projected R-version support.
+[^1]: Note that we only update the required version on package release, so you may see older versions listed in published CRAN packages.
```
Comment (jennybc) [inline content/blog/2019/r-version-support/index.md:47]: If we want to be really pedantic, we could spell this out more. I.e. we don't rush to re-release all packages after a minor version increment of R itself, but rather we bump the minimum version of R at the next obvious opportunity.

**Review by jennybc — APPROVED** [review]

File: content/blog/2019/r-version-support/index.md (hunk 2 above)
Comment (hadley) [reply content/blog/2019/r-version-support/index.md]: While you were looking at this I made some more changes. What do you think of the wording now?

File: content/blog/2019/r-version-support/index.md [hunk 3]
```diff
@@ -15,36 +15,24 @@ rmd_hash: efe6168c59ef5121
 
 ---
 
-## Which versions of R do tidyverse packages support?
+Our general policy is to support the **current version**, the **devel version**, and **four previous versions** of R. This policy applies to all packages that we support, i.e. as well as all tidyverse packages, it also applies to the infrastructure packages that we maintain in [r-lib](https://github.com/r-lib).
 
-Though package-specific minimum versions of R are given in the `Depends` fields of individual `DESCRIPTION` files, our general policy is to support the **current version**, the **devel version**, and **four previous versions** of R.
+This, the *official* minimum supported versions of R for the tidyverse are as described here:
```
Comment (jennybc) [inline content/blog/2019/r-version-support/index.md:20]: I feel like the comma is weird. Should it go away? Get a matching friend? Or do you mean "Thus"?

File: content/blog/2019/r-version-support/index.md [hunk 4]
```diff
@@ -15,36 +15,24 @@ rmd_hash: efe6168c59ef5121
...
+| 2020-04-24 | 4.0                | 3.3                       |
+| 2021-05-18 | 4.1                | 3.4                       |
+| 2022-04-22 | 4.2                | 3.5                       |
+| 2023-04-21 | 4.3                | 3.6                       |
+| 2024-04-24 | 4.4                | 4.0                       |
+| 2025       | 4.5                | 4.1                       |
+| 2026       | 4.6                | 4.2                       |
 
-Over the past several years and moving forward (as described in the "Release plans" section of the [R Developer Page](https://developer.r-project.org/)), R version releases occur annually:
+Note that, as described in the "Release plans" section of the [R Developer Page](https://developer.r-project.org/), R version releases occur annually in Spring. We generally update the required version in package metadata on the next package release after the R package, so you may see older versions listed in published CRAN packages.
```
Comment (jennybc) [inline content/blog/2019/r-version-support/index.md:33]: This doesn't seem right:

"on the next package release after the R package"

After the R release?

**Review by jennybc — APPROVED** [review]

**Pull request merged**